src/promptflow-devkit/promptflow/_sdk/_service/apis/connection.py

```diff
@@ -77,10 +77,6 @@ def _get_connection_operation(working_directory=None):
 class ConnectionList(Resource):
     @api.doc(parser=working_directory_parser, description="List all connection")
     @api.marshal_with(list_connection_field, skip_none=True, as_list=True)
-    @local_user_only
```

Comment:
@lalala123123  to double-confirm here

Reply:
why we remove the local user only tag?
If this is for extension, they can set an extra header in request to pass this check

Reply:
offline sycned, only keep header in ConnectionWithSecret